exercise-8/pages/index.vue

```diff
@@ -1,11 +1,160 @@
 <template>
+  <div>
+    <nav class="navbar navbar-expand-lg bg-dark navbar-dark">
```

Comment:
This is one of the semantic HTML tags I was expecting but the "divitis" below :point_down: makes me not give you the :rocket: for this optional task. Sorry.